### DISCUSSION TITLE
Reply / Resolve buttons confusing

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#53958] [Mobile] [answered]
GitHub Mobile beta

In this thread, I clicked the wrong button:
![Screenshot_20230428-083406.png](https://user-images.githubusercontent.com/2119212/235150524-dc31fa0e-9c7f-478d-b6f2-33092d9e897f.png)

(You can't see that because the GitHub Mobile version apparently doesn't show the resolve+unresolve actions.)

Please consider adding an icon next to `Resolve`. I'm hoping that'd decrease the likelihood of people like me clicking the wrong button.

**Comment by taki-on** [comment, marked as answer]
Thank you for your report. An icon for the button will be better, indeed! We will discuss it internally. Thanks!
